src/libNode/Node.h

```diff
@@ -239,10 +239,10 @@ class Node : public Executable, public Broadcastable
     bool ProcessFinalBlock(const std::vector<unsigned char>& message,
                            unsigned int offset, const Peer& from);
     bool ProcessForwardTransaction(const std::vector<unsigned char>& message,
-                                   unsigned int offset, const Peer& from);
+                                   unsigned int cur_offset, const Peer& from);
```

Comment:
Although it is automated, I prefer it to be without _ for consistency with other code. same for block_num.

Reply:
The name is taken from the definition of `ProcessForwardTransaction`.

https://github.com/Zilliqa/Zilliqa/blob/70fbb16b9852844b21c9fcc3c081e2653c2b2fc5/src/libNode/FinalBlockProcessing.cpp#L1197-L1199